include/zephyr/drivers/video.h

```diff
@@ -1338,6 +1338,15 @@ int video_estimate_fmt_size(struct video_format *fmt);
  */
 #define VIDEO_PIX_FMT_GREY VIDEO_FOURCC('G', 'R', 'E', 'Y')
 
+
+/**
+ * @code{.unparsed}
+ *   0          1          2          3          3 2 1 0
+ * | 0000Yyyy | 0000Yyyy | 0000Yyyy | 0000Yyyy | 0000Yyyy | ...
+ * @endcode
+ */
+#define VIDEO_PIX_FMT_Y4      VIDEO_FOURCC('Y', '0', '4', ' ') /*  4  Greyscale     */
```

Comment:
Linux adds comment at the end of the pixel format, but they are not rendered in Doxygen so can be removed from Zephyr. Also you may remove the spaces after `Y4`...

```
#define VIDEO_PIX_FMT_Y4 VIDEO_FOURCC('Y', '0', '4', ' ')
```